genie-web/src/main/java/com/netflix/genie/GenieWeb.java

```diff
@@ -41,8 +41,8 @@
 @EnableAspectJAutoProxy
 public class GenieWeb {
 
-    protected static final String SPRING_CONFIG_LOCATION = "spring.config.location";
-    protected static final String USER_HOME_GENIE = "${user.home}/.genie/";
+    static final String SPRING_CONFIG_LOCATION = "spring.config.location";
+    static final String USER_HOME_GENIE = "${user.home}/.genie/";
```

Comment:
Will try to keep this in mind for the future. For now this is already squashed so too late